packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -365,4 +365,64 @@ StructuredListMedia.args = {
       selected: false
     },
   ]
+}
+
+
+/*<------------------------------------ Select ---------------------------------------->*/
+
+const StructuredListSelectableTemplate = ({ listItems, striped, showHeader, headerId, context, sx }) => {
+  return ` 
+       <cbp-structured-list
+          ${striped ? `striped` : ''}
+          selectable
+          header-id="list-header"
+          ${context && context != 'light-inverts' ? `context=${context}` : ''}   
+          ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+        >
+        ${showHeader ? `<div slot="cbp-structured-list-header" id="${headerId}"><cbp-checkbox value='1'> <input type='checkbox' name='checkbox' value='1' /></cbp-checkbox> <span>XX results - Y filters Applied Updated: XX/XX/XXXX XX:XX:XX EST</span></div>` : ''}
+    
+        ${generateItems(listItems)}
+
+        
+        </cbp-structured-list>
+      `;
+};
+export const StructuredListSelectable = StructuredListSelectableTemplate.bind({});
+StructuredListSelectable.argTypes = {
+  listItems: {
+    description: 'Configure various aspects of the list items within the structured list.',
+    control: 'object',
+  },
+  // selectable: true
+}
+StructuredListSelectable.args = {
+  showHeader: false,
+  listItems: [
+    {
+      content: "<cbp-checkbox value='1'> <input type='checkbox' name='checkbox' value='1' /></cbp-checkbox> <span>Structured List Selectable Item 1</span>",
```

Comment:
You need labels for each of the checkboxes. Each of the values should be different (make them sequential?).

And finally because you're passing the checkbox as content, you can't pass in the context from the parent. If I change the context to light-always or dark-always, the checkboxes remain in their current state and don't match the structured list context. I think this probably requires modifying your story and adding a new function to generate selectable list items.

Reply:
wrote new function generateSelectableItems to address this. is applying context of parent to checkbox as well as adding the 'hidden' labels for the checkbox